pkg/ingress/enhanced_backend_builder.go

```diff
@@ -164,11 +164,7 @@ func (b *defaultEnhancedBackendBuilder) buildConditions(_ context.Context, ingAn
 	if err != nil {
 		return nil, err
 	}
-	for _, condition := range conditions {
```

Comment:
Please keep this validation block in place. We've seen feature requests to optionally disable validation webhook for ingress. There would also be a case where the invalid ingress already exists before the controller got updated. We still want to error out from model builder if validation fails.

Reply:
I do not think we should support disabling our validation webhook for ingress. If someone disables it, they can deal with the fallout of configurations the webhook would prevent.

We should not add unnecessary complexity.

Reply:
As for me I'm not a fun of duplication of logic/checkings etc, so I'd prefer to keep validation in 1 place. @kishorj @johngmyers can we meet consolidate decision(maybe by voting or etc.)?

Reply:
We discussed further on this matter in the provider-aws meeting.  Since there is a possibility of having ingress resources from a prior version (before the webhook validation), we still do these validation for 2.5.0. We can clean up the validation in future releases.

Reply:
Validation returned back